src/component/form-fields.html

```diff
@@ -1,10 +1,17 @@
 <template>
 
-  <form-field
+  <form-field repeat.for="element of schema"
+    if.bind="hasKeyProp(element)"
     value.bind="model[element.key]"
     message.bind="messages[element.key]"
-    element.bind="element"
-    repeat.for="element of schema">
+    element.bind="element">
+  </form-field>
+
+  <form-field repeat.for="element of schema"
+    if.bind="!hasKeyProp(element)"
+    value.bind="model"
+    message.bind="messages[element.key]"
+    element.bind="element">
```

Comment:
I have tried this yet it does not work when doing two way databinding on primitives.